meta-ostro/classes/ostro-image-extensions.bbclass

```diff
@@ -0,0 +1,15 @@
+# Class that provides additional tools and methods for image conversions.
```

Comment:
patch for those submitted upstream, but not sure how long it would take to accept it.
My idea was that after functionality verification, I'll move bmap releated things into separate class where image-dsk would be inheriting from. vdi would stay in image-dsk, and all checksum related things would go to upstream. meanwhile, it can be in ostro distro configs (not even needed to be in bbclass to be available globally).